dace/codegen/targets/cpu.py

```diff
@@ -1285,12 +1286,33 @@ def _generate_Tasklet(self, sdfg, dfg, state_id, node, function_stream,
                               node)
         callsite_stream.write('{', sdfg, state_id, node)
         callsite_stream.write(inner_stream.getvalue(), sdfg, state_id, node)
+        callsite_stream.write(after_memlets_stream.getvalue())
         callsite_stream.write('}', sdfg, state_id, node)
         callsite_stream.write(outer_stream_end.getvalue(), sdfg, state_id, node)
 
         self._dispatcher.defined_vars.exit_scope(node)
 
-        callsite_stream.write(after_memlets_stream.getvalue())
+
+    def unparse_tasklet(self, sdfg, state_id, dfg, node, function_stream,
+                        inner_stream, locals, ldepth, toplevel_schedule):
+        # Call the generic CPP unparse_tasklet method
+        unparse_tasklet(sdfg, state_id, dfg, node, function_stream,
```

Comment:
to be clearer, shall we just import `cpp` and call `cpp.unparse_tasklet` here?
Minor comment though

Reply:
Yeah I agree, then I just have to track down every call to `cpp` in the code :wtfjoj: This is why `import *` sucks balls

Reply:
Went through every function in `cpp.py` and replaced calls in `cpu.py` :jojpride: